client/src/pages/Login.js

```diff
@@ -0,0 +1,235 @@
+import React, { Component } from "react";
+
+import {
+  Typography,
+  Grid,
+  Paper,
+  Button,
+  Box,
+  TextField,
+  Snackbar,
+} from "@material-ui/core";
+import MuiAlert from "@material-ui/lab/Alert";
+import { withStyles } from "@material-ui/core/styles";
+import { Route, Link } from "react-router-dom";
+import pic1 from "../assets/pic1.png";
+import pic2 from "../assets/pic2.png";
+import pic3 from "../assets/pic3.png";
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+const width = window.innerWidth;
```

Comment:
Okay you shouldn't be using this variable for the width in the component... because it gets initialized only once and is not best practice

Please look at: https://material-ui.com/components/use-media-query/
For class-based components, also look at: https://material-ui.com/customization/breakpoints/

You should also be using the theme breakpoints in Material-ui (not hardcoded values for the media query)

Reply:
Use breakpoints to assign different widths to the image